include/antler/project/location.hpp

```diff
@@ -22,12 +25,18 @@ namespace antler::project::location {
 /// @note requires `gh` is installed and user is authenticated or the repo is public.
 /// @param l Location to evaluate.
 /// @return true if l is a repo, calls `gh` to test.
-[[nodiscard]] bool is_github_org_repo_shorthand(std::string_view l);
+[[nodiscard]] bool is_github_shorthand(std::string_view l);
 /// @param l  Location to evaluate.
 /// @return true if l is reachable.
 bool is_reachable(std::string_view l);
 /// @param l  Location to evaluate.
 /// @return true if l looks like a url.
 bool is_url(std::string_view l);
 
+bool clone_github_repo(const std::string& org, const std::string& repo, const std::string& branch, uint32_t jobs, system::fs::path dest = ".");
+bool clone_git_repo(const std::string& url, const std::string& branch, uint32_t jobs, system::fs::path dest = ".");
+bool pull_git_repo(system::fs::path src);
+std::string github_request(const std::string& org, const std::string& repo);
+std::string get_github_default_branch(const std::string& org, const std::string& repo);
```

Comment:
Shouldn't these be marked `nodiscard`? 
```suggestion
[[nodiscard]] bool clone_github_repo(const std::string& org, const std::string& repo, const std::string& branch, uint32_t jobs, system::fs::path dest = ".");
[[nodiscard]] bool clone_git_repo(const std::string& url, const std::string& branch, uint32_t jobs, system::fs::path dest = ".");
[[nodiscard]] bool pull_git_repo(system::fs::path src);
[[nodiscard]] std::string github_request(const std::string& org, const std::string& repo);
[[nodiscard]] std::string get_github_default_branch(const std::string& org, const std::string& repo);
```

Reply:
Fixed in 1e42809

Reply:
Further correction in 03260ee45452415776f21f6d6eb25ecd69704139